internal/common/voting/election.go

```diff
@@ -55,8 +55,90 @@ func (e *Election) CloseBallot() shared.ClientID {
 }
 
 func (e *Election) bordaCountResult() shared.ClientID {
-	// TODO implement Borda count winner selection method.
-	return e.pluralityResult()
+	// Implement Borda count winner selection method
+	//(may need to modify if methods design is changed)
+	var candidatesNumber int = 0
+	var islandsNumber int
+	var votesLayoutElect map[int][]int
+	votesSliceSquare := e.votes
+	candidatesNumber = len(e.islandsToVote)
+	islandsNumber = len(votesSliceSquare)
+	scoreInit := candidatesNumber + 1
+
+	//Transfer e.votes to a preference map with type "int"
+	for i := 0; i < islandsNumber; i++ {
+		for j := 0; j < candidatesNumber; j++ {
+			for k := 0; k < candidatesNumber; k++ {
+				if votesSliceSquare[i][j] == e.islandsToVote[k] {
+					votesLayoutElect[i][k] = scoreInit
+					scoreInit = scoreInit - 1
+					break
+				}
+			}
+		}
+	}
+
+	//Sort the preference map in order.
+	var order []int
+	var index []int
+	var score []int
+	preferenceMap := make(map[int][]int)
+	for k, v := range votesLayoutElect {
+		j := 0
+
+		for t := 0; t < candidatesNumber; t++ {
+			order[t] = v[t]
+		}
+
+		for i := 0; i < candidatesNumber; i++ {
+
+			maxlim := 100
```

Comment:
Try to avoid initializing things with "magic numbers" (so arbitrary constants with no explanation) but they're not always easily avoidable

Reply:
set a maxlimitation randomly just to sort the data in preference list from small to large, I'll try to avoid this.

Reply:
fixed